### DISCUSSION TITLE
提交检验

### DISCUSSION THREAD

**Discussion by wjwjwjw** [alibaba/formily#1339] [Q&A 2.x] [answered]
使用弹窗组件，自定义底部按钮，提交时校验，校验不通过不可提交，校验通过获取表单项的值，如何实现

**Comment by janryWang** [comment, marked as answer]
https://github.com/alibaba/formily/discussions/1303

**Reply by wjwjwjw** [reply]
> #1303

这里有两个疑问，弹窗组件会默认创建form实例是么？怎样拿到表单的values呢？

**Reply by janryWang** [reply]
内部会创建一个实例，拿values，.then就能拿，文档看仔细点

**Reply by wjwjwjw** [reply]
> 内部会创建一个实例，拿values，.then就能拿，文档看仔细点

懂了，谢大佬耐心解答

**Reply by AstraeusSEA** [reply]
> 内部会创建一个实例，拿values，.then就能拿，文档看仔细点

请问知道提交时怎么触发自定义组件里面内容的校验吗
